pvlib/pvsystem.py

```diff
@@ -536,7 +536,7 @@ def pvsyst_celltemp(self, poa_global, wind_speed, temp_air):
         kwargs = _build_kwargs(['eta_m', 'alpha_absorption'],
                                self.module_parameters)
         return pvsyst_celltemp(poa_global, wind_speed, temp_air,
-                               temp_model=self.racking_model, **kwargs)
+                               loss_factors=self.racking_model, **kwargs)
```

Comment:
I'm not a fan of this renaming.  Although `loss_factors` may be more in line with PVsyst terms, I think using a term that is common with the SAPM temperature model is beneficial.  The `temp_model` variable name aligns with the `TEMP_MODELS` constant dict in `pvlib.pvsystem`.  I'm open to a term other than `temp_model` (which I don't object to) but only if it applies to all temperature models.

Reply:
`sapm_celltemp` simply calls this keyword argument `model`. I like `model` or `loss_factors` or `heat_loss_factors` more than `temp_model`. I think `temp_model` could apply to the higher level `sapm_celltemp` vs. `pvsyst_celltemp`, but not here.

Reply:
I'd prefer the same keyword and behavior in both functions.  Either a string which maps to an entry in `TEMP_MODELS`, a dict, or (cringe) a list or tuple.  I'm OK with `model` but `loss_factors` and variants only describe coefficients for the PVsyst model, the SAPM coefficients have different units.

Reply:
`loss_factors` is more in line with generic heat transfer, at least that's what motivated my suggestion.  I dislike the `*model*` variations because in my view the numbers represent model *parameters*, not models.

Reply:
I am using the term model to refer to the vector of values, in the same way pvlib uses the keyword `model` for a vector of values for a DC model.  Open to other ideas for naming.

Reply:
I'd be fine with that.

Reply:
I wouldn't.  alpha and eta_m are (should be) module specific, where as U0 and U1 are more tied to the mounting arrangement.

Reply:
Name changes in [commit](https://github.com/pvlib/pvlib-python/pull/651/commits/c37cbf6cccb9d1c076aec5553a612a60e5707afb) are OK with me